src/app/sign-up/sign-up-get-starter-bitclout/sign-up-get-starter-bitclout.component.html

```diff
@@ -15,6 +15,27 @@
     <div class="fs-18px mt-15px mb-15px">BitClout is a cryptocurrency that's required to use the site.</div>
   </div>
 
+  <div *ngIf="!jumioError && showJumio">
```

Comment:
I'm confused, if they come back from Identity won't they see this?

Reply:
ahh I just need to kill this - I originally had it as part of the app flow but then diamondhands wanted it in identity so it was available to 3rd party nodes